.github/workflows/build-test.yaml

```
@@ -2623,21 +2623,23 @@ jobs:
             exit 1
           fi
 
-          if [ "$(kubectl get all -n postgres-test --ignore-not-found | wc -l | tr -d '\n')" == "0" ]; then
-            printf "postgres-test namespace does not contain any resources\n\n"
-            kubectl get all -n postgres-test
-            exit 1
-          fi
+          for ns in "${APP_NAMESPACES[@]}"; do
+            if [ "$(kubectl get all,secrets,configmap,pvc -n "$ns" -l "kots.io/app-slug=$APP_SLUG" --ignore-not-found | wc -l | tr -d '\n')" == "0" ]; then
+              printf "%s namespace does not contain any resources\n\n" "$ns"
+              kubectl get all,secrets,configmap,pvc -n "$ns"
+              exit 1
+            fi
+          done
 
-          if [ "$(kubectl get all -n redis-test --ignore-not-found | wc -l | tr -d '\n')" == "0" ]; then
-            printf "redis-test namespace does not contain any resources\n\n"
-            kubectl get all -n redis-test
+          if [ "$(kubectl get rabbitmqclusters.rabbitmq.com -n rabbitmq-system --ignore-not-found | wc -l | tr -d '\n')" == "0" ]; then
```

this shouldn't have `--ignore-not-found`, right?
```suggestion
          if [ "$(kubectl get rabbitmqclusters.rabbitmq.com -n rabbitmq-system | wc -l | tr -d '\n')" == "0" ]; then
```

---

actually, nevermind, the count would be `0` in that case.